arcgis-ios-sdk-samples/Features/Add delete related features/RelatedFeaturesViewController.swift

```diff
@@ -130,15 +130,19 @@ class RelatedFeaturesViewController: UITableViewController {
         // show progress hud
         SVProgressHUD.show(withStatus: "Applying edits")
         
-        relatedTable.applyEdits { [weak self] (_, error) in
+        relatedTable.applyEdits { [weak self] (results, error) in
             SVProgressHUD.dismiss()
             
-            if let error = error {
-                // show error
+            if let firstResult = results?.first {
+                if firstResult.completedWithErrors {
```

Comment:
Again I'm checking the flag rather than doing conditional unwrapping here because it's a better pattern if there are multiple results.